code_1.py

```diff
@@ -6,22 +6,24 @@
 from adafruit_hid import find_device
 from hid_gamepad import Gamepad
 import usb_hid
+import usb_cdc
 
 trellis = adafruit_trellism4.TrellisM4Express()
 
+serial = usb_cdc.data
 gp = Gamepad(usb_hid.devices)
-gp.press_buttons(2)
 
-"""
-while True:
-    trellis.pixels[0, 0] = (0, 0, 0)
-    gp.press_buttons(2)
-    time.sleep(0.5)
-    trellis.pixels[0, 0] = (0, 0, 0)
-    gp.release_buttons(2)
-    time.sleep(0.5)
-"""
+if serial is None:
+    print("No serial connection!")
+else:
+    print("Serial present")
+    serial.timeout = 0.1
+
+def clearLeds(trellis):
+    trellis.pixels.fill(0)
 
+def inRange(value, min, max):
```

Comment:
⭐ I think it would be more pythonic if the max was exclusive given how `range` generally works. Or maybe name it explicitly `inRangeInclusive`